the_board/lib/pages/workroomPage.dart

```diff
@@ -0,0 +1,378 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/cupertino.dart';
+
+class WorkroomPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      initialRoute: '/',
```

Comment:
can we change the name of the route from / to something like /chatsList (or similar)?